contracts/AnchoredPriceView.sol

```diff
@@ -221,6 +225,9 @@ contract AnchoredPriceView {
      * @return The price for the given cToken address
      */
     function getUnderlyingPrice(address cToken) external view returns (uint256) {
+        if (breaker == true) {
+            return anchor.getUnderlyingPrice(cToken);
```

Comment:
so, if breaker is on, price in ETH is returned, and if it's off - USD?

Reply:
this is true, here, but in anticipation of going for only ETH, right?